source/form.html

```diff
@@ -0,0 +1,121 @@
+                    <input class="form__input" type="text" id="last-name" name="last-name" placeholder="Иванов">
+                    <label class="form__label" for="middle-name">Отчество:</label>
+                    <input class="form__input" type="text" id="middle-name" name="middle-name" placeholder="Александрович">
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Контактная информация:</legend>
+                    <label class="form__label" for="user-phone">Контактный телефон*:</label>
+                    <input class="form__input" type="tel" id="user-phone" name="phone" placeholder="Введите телефон">
+                    <label class="form__label" for="user-email">Электронная почта*:</label>
+                    <input class="form__input" type="email" id="user-email" name="email" placeholder="Введите e-mail">                    
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Ваше общее впечатление:</legend>
+                    <label class="form__label form__label--bold" for="pisitive">Скорее положительное</label>
+                    <input class="form__input" type="radio" id="positive" name="impression" checked>
+                    <label class="form__label form__label--bold" for="negative">Скорее отрицательное</label>
+                    <input class="form__input" type="radio" id="negative" name="impression">
+                    <label class="form__label form__label--bold" for="confused">Затрудняюсь ответить</label>
+                    <input class="form__input" type="radio" id="confused" name="impression">
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Посещенные достопримечательности:</legend>
+                    <input class="form__input" type="checkbox" id="bridge" name="bridge" checked>
+                    <label class="form__label form__label--bold" for="bridge">Мост Дьявола</label>
+                    <input class="form__input" type="checkbox" id="mountain" name="mountain" checked>
+                    <label class="form__label form__label--bold" for="mountain">Гора-Колокол</label>
+                    <input class="form__input" type="checkbox" id="park" name="park" checked>
+                    <label class="form__label form__label--bold" for="park">Слайд-парк</label>
+                    <input class="form__input" type="checkbox" id="rocks" name="rocks">
+                    <label class="form__label form__label--bold" for="rocks">Слайд-парк</label>
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Опишите свои эмоции:</legend>
+                    <textarea class="form__textarea" name="user-emotions" id="emotions" cols="30" rows="10" placeholder="Опишите подробно все свои восторги "></textarea>
```

Comment:
what for  `name` attr for textarea?

Reply:
have read the theory about textarea tag. Yep, we can ommite id here, because tag textarea is wrapped by form. The name attribute is required because the form data will be submitted to the server and the textarea data needs to be identified. is it so?

Reply:
honestly, I don't know) let it be